docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -1,21 +1,35 @@
 = Scaling and Upgrading a Hazelcast Deployment
 
-To scale an existing Hazelcast deployment, use the following command:
+To scale an existing Hazelcast deployment, modify the yaml definition you used to deploy Hazelcast cluster by updating `clusterSize` field and re-apply it again.
 
-[source,shell]
+Assuming the following is the existing configuration:
+
+[source,yaml]
 ----
-kubectl scale hazelcast <name> --replicas <replica-count>
+apiVersion: hazelcast.com/v1alpha1
```

Comment:
Could you add `kubectl scale` option as well? It will be available with 5.11 release.

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/150/commits/8854a2f819706699a1324f037a487a540fe70da3